AirportDepartures.playground/Contents.swift

```diff
@@ -40,9 +144,31 @@ import UIKit
 //: c. Make your `FlightStatus` enum conform to `String` so you can print the `rawValue` String values from the `enum`. See the [enum documentation](https://docs.swift.org/swift-book/LanguageGuide/Enumerations.html).
 //:
 //: d. Print out the current DepartureBoard you created using the function
-
-
-
+func printDeparture(departureBoard: DepartureBoard) {
+    
+    let dateFormat = DateFormatter()
+    dateFormat.timeStyle = .short
+    dateFormat.dateStyle = .none
+    
+    for departure in departureBoard.departureFlights {
+        
+        var time = ""
+        var terminal = ""
+        
+        if let departureTime = departure.departureTime {
+            
+            time = dateFormat.string(from: departureTime)
+        }
+        
+        if let terminalNum = departure.terminal {
+            terminal = "\(terminalNum)"
+        }
+        print("\nDestination \(departure.airport.destination) \nAirline: \(departure.airline) Flight: \(departure.flightNumber) Departure Time: \(time) Terminal: \(terminal) Flight Status: \(departure.fightStatus)")
+    }
```

Comment:
Nice use of variables that are providing default values in case you can't unwrap the optionals. Very good approach.